### ISSUE TITLE
No AuthResponse when Port is already in use

### ISSUE THREAD

**Issue by Mrnikbobjeff** [JohnnyCrazy/SpotifyAPI-NET#38]
There is a  Try/Catch block in the SimpleHttpServer class in the Listen Method where the exception is simply ignored. This poses an issue when Port 80 is already in use or was used before in another instance of the SpotifyAPI-NET library which was cancelled while the app was still waiting on a response. While this can be handled on the user side by closing the server before the application exits, I do believe that this should at least throw an exception which at least hints at the cause for the authentication's failure. This was also the problem in my previous Issue, but as it was closed I will reopen another Issue here.

**Comment by JohnnyCrazy** [comment]
Implemented in 006f085c4b100be4042a13e729747efad09273d6

In the long-term, I maybe need to implement some kind of BackgroundWorker, since I currently just terminate the thread which produces error code 10004 (which is also ignored now)
